manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: istiod-clusterrole{{- if not (eq .Values.revision "")}}-{{ .Values.revision }}{{- end }}-{{ .Values.global.istioNamespace }}
```

Comment:
I'm not sure if istioNamespace is still needed here - we have revision. 

If we want to keep it - I think revision should be at the end.

Reply:
Removed, thanks